functions/wifi.bash

```diff
@@ -9,12 +9,9 @@ wifi_setup() {
     echo "FAILED"; return 1
   fi
   if [ -n "$INTERACTIVE" ]; then
-    SSID=$(whiptail --title "Wifi Setup" --inputbox "Which Wifi (SSID) do you want to connect to?" 10 60 3>&1 1>&2 2>&3)
-    if [ $? -ne 0 ]; then return 1; fi
-    PASS=$(whiptail --title "Wifi Setup" --inputbox "What's the password for that Wifi?" 10 60 3>&1 1>&2 2>&3)
-    if [ $? -ne 0 ]; then return 1; fi
-    WIFICOUNTRY=$(whiptail --title "Wifi Setup" --inputbox "Please enter the two-letter country code matching your region eg. US DE NZ AU...\nSee https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2" 10 60 3>&1 1>&2 2>&3)
-    if [ $? -ne 0 ]; then return 1; fi
+    if ! SSID=$(whiptail --title "Wifi Setup" --inputbox "Which Wifi (SSID) do you want to connect to?" 10 60 3>&1 1>&2 2>&3); then return 1; fi
```

Comment:
Have you tested this rewrite on actual hardware installation?

Reply:
This is a general question actually. When we touch so much code we must verify that a "real" installation works.

Reply:
No haven't checked this specific one because it's rather obvious that it does not change functionality.
We can make a real HW test by the end of the review process.

Reply:
Sounds fair. Let's do it in the end.